devTools/Dockerfile-php80-xdebug

```diff
@@ -25,8 +20,16 @@ RUN set -eux; \
     apk add --no-cache --virtual .phpexts-rundeps ${runDeps}; \
     apk del .build-deps
 
+RUN apk add --no-cache \
+        ncurses \
+        make \
```

Comment:
why do we need to have `make` inside container? I didnt' see that we use it elsewhere except on host machine.

Reply:
To run the whole set of E2E tests with benchmarks and stuff: `make test-e2e`
(It is possible to do without `make`, but quite a hassle.)